tests/modules/accounting/test_bunker.py

```diff
@@ -184,6 +184,17 @@ def bunker(web3, lido_validators) -> BunkerService:
         '0x04': simple_key('0x04'),
         '0x05': simple_key('0x05'),
     }
+    # service.w3.cc.spec = ConsensusSpecRequiredOnly(
+    #     MIN_VALIDATOR_WITHDRAWABILITY_DELAY="256",
+    #     EPOCHS_PER_SLASHINGS_VECTOR="8192",
+    #     PROPORTIONAL_SLASHING_MULTIPLIER_BELLATRIX="3",
+    #     EFFECTIVE_BALANCE_INCREMENT="1000000000",
+    #     MAX_EFFECTIVE_BALANCE="32000000000",
+    #     MIN_PER_EPOCH_CHURN_LIMIT="4",
+    #     CHURN_LIMIT_QUOTIENT="65536",
+    #     ETH1_ADDRESS_WITHDRAWAL_PREFIX="0x01",
+    #     MAX_WITHDRAWALS_PER_PAYLOAD="16",
+    # )
```

Comment:
?

Reply:
will be fixed when we approve the concept because it brings a lot of issues in tests